packages/slice-machine/lib/models/common/Environment.ts

```diff
@@ -9,6 +9,11 @@ export interface UpdateVersionInfo {
   packageManager: "npm" | "yarn";
   updateCommand: string;
   updateAvailable: boolean;
+  availableVersions: {
```

Comment:
It should be called availableUpdates.
AvailableVersions is too global for the data we're trying to form.
Also, each key should be potentially undefined.

If there is no patch, minor or major then this object should be empty.

Reply:
`UpdateVersionInfo.availableUpdates` is too global?